css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -35,30 +35,52 @@ main {
  * .box--top-left -> This element should be be placed in the top, left corner.
 */
 
+.box {
+  position: absolute;
+}
+
 .box--top-left {
   background-color: #aed6f1;
+  left: 0;
+  top: 0;
 }
 
 .box--top-center {
   background-color: #85c1e9;
+  top: 0;
+  left: 50%;
+  margin-left: -40px;
```

Comment:
nice that it worked
I didn't know how to solve it so I googled it & did "transform: translateX(-50%)"; that worked too